web/migrations/0051_achievement_badge_type_achievement_teacher_and_more.py

```
@@ -0,0 +1,69 @@
+            field=models.ForeignKey(
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
+                related_name="awarded_by",
+                to=settings.AUTH_USER_MODEL,
+            ),
+        ),
+        migrations.AddField(
+            model_name="profile",
+            name="wallet_address",
+            field=models.CharField(
+                blank=True, help_text="Student's Ethereum wallet address for receiving NFT badges", max_length=42
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)